src/components/Explore/TokenDetails/ShareButton.tsx

```diff
@@ -20,64 +23,47 @@ const ShareButtonDisplay = styled.div`
     color: ${({ theme }) => darken(0.1, theme.textSecondary)};
   }
 `
+
+const Share = styled(ShareIcon)<{ open: boolean }>`
+  width: 18px;
+  height: 18px;
+  :hover {
```

Comment:
If this is hover state for the share icons, can you also make this the hover state for favoriting the token? it prob should be the same as hover state for share